### ISSUE TITLE
Creating disk containing a dot (.) incomplete

### ISSUE THREAD

**Issue by khenderick** [openvstorage/framework#912]
Creating disk containing a dot (.) doesn't work as expected, according @grimpy. It will be visible in de GUI, but not available through other means (fuse, edge, ...)

**Comment by khenderick** [comment]
When in the GUI a vDisk name is entered, we allow the customer to use any name, and try to convert it to a reasonable filename that can be used by the volumedriver. However, it "detects" extensions (name ending in a dot and 3 or 4 characters), and will then use that exact filename. However, the volumedriver only accepts `.raw` files.

We need to change the code so it always ends with `.raw`.

**Comment by khenderick** [comment]
Fixed by #982, packaged in openvstorage-2.7.3-rev.4061.083329a

**Comment by JeffreyDevloo** [comment]
### Steps
- Installed latest packages
- Installed cluster
- Added roles to disks
- Added backend
- Initialized disks and claimed asds
- Added usable preset
- Added vpool on node 1
- Added vdisks with dots in their name

### Output

**All disks have the .raw extension**

```
root@ovs-node1:~# ls /mnt/myvpool01/
test1.raw  test2.trailingdot..raw  test3.qcow2.raw
```

**Detailed view**
![selection_001](https://cloud.githubusercontent.com/assets/17570109/19440763/63b58be2-9483-11e6-9b0a-b0f691fafbae.png)

### Test result

Test passed.

### Packages
- openvstorage                       2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.388.1f4375d-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage Web Applications
